cmd/main.go

```diff
@@ -119,11 +122,22 @@ func main() {
 
 	// Subscribe to cache update
 	if !*bigqueryOnly {
-		err = s.SubscribeBranchCacheUpdate(
+		sub, err := s.SubscribeBranchCacheUpdate(
 			ctx, *storeProject, branchCacheVersionBucket, subscriberPrefix, pubsubTopic)
 		if err != nil {
 			log.Fatalf("Failed to subscribe to branch cache update: %v", err)
 		}
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
Some comments on what you're doing here?

Reply:
Done